admin/sql/updates/20160604-mbs-8951.sql

```diff
@@ -0,0 +1,4 @@
+\set ON_ERROR_STOP 1
+
+DROP INDEX CONCURRENTLY edit_note_idx_post_time_edit;
+CREATE INDEX CONCURRENTLY edit_note_idx_post_time_edit ON edit_note (post_time DESC NULLS LAST, edit DESC);
```

Comment:
Good idea. Created http://tickets.musicbrainz.org/browse/MBS-8973 (feel free to update that as necessary).